Packs/CortexAttackSurfaceManagement/Scripts/InferWhetherServiceIsDev/InferWhetherServiceIsDev_test.py

```diff
@@ -81,34 +81,82 @@ def test_full_truth_table():
     # Blank list means no external classification or tag matches.
     sample_no_match = []
     sample_dev_classification = ["DevelopmentEnvironment"]
+    sample_dev_hierarchy = ["ENG-DEV"]
+    sample_prod_hierarchy = ["ENG-PROD"]
 
     from InferWhetherServiceIsDev import final_decision
 
     # dev == True, all else is False
 
     # kv pair contains no indicators
     # DevEnv is set (--> dev)
-    assert final_decision(sample_dev_classification, sample_no_match, sample_no_match)["result"]
+    assert final_decision(sample_dev_classification, sample_no_match, sample_no_match,
+                          sample_no_match, sample_no_match, "")["result"]
     # DevEnv is not set (--> can't tell)
-    assert not final_decision(sample_no_match, sample_no_match, sample_no_match)["result"]
+    assert not final_decision(sample_no_match, sample_no_match, sample_no_match, sample_no_match, sample_no_match, "")["result"]
 
     # kv pair contains dev indicators only
     # DevEnv is set (--> dev)
-    assert final_decision(sample_dev_classification, sample_dev_tag, sample_no_match)["result"]
+    # Dev Tags only
+    assert final_decision(sample_dev_classification, sample_dev_tag, sample_no_match,
+                          sample_no_match, sample_no_match, "")["result"]
+    # Dev Hierachy only
+    assert final_decision(sample_dev_classification, sample_no_match, sample_no_match,
+                          sample_dev_hierarchy, sample_no_match, "")["result"]
+    # Both Dev Tags and Hierarchy
+    assert final_decision(sample_dev_classification, sample_dev_tag, sample_no_match,
+                          sample_dev_hierarchy, sample_no_match, "")["result"]
+    #
     # DevEnv is not set (--> dev)
-    assert final_decision(sample_no_match, sample_dev_tag, sample_no_match)["result"]
+    # Dev Tag only
+    assert final_decision(sample_no_match, sample_dev_tag, sample_no_match, sample_no_match, sample_no_match, "")["result"]
+    # Dev Hierachy only
+    assert final_decision(sample_no_match, sample_no_match, sample_no_match, sample_dev_hierarchy, sample_no_match, "")["result"]
+    # Both Dev Tags and Hierarchy
+    assert final_decision(sample_no_match, sample_no_match, sample_no_match, sample_dev_hierarchy, sample_no_match, "")["result"]
```

Comment:
i think this was intended to have `sample_dev_tag` as the second arg

Reply:
fixed in [PR feedback](https://github.com/demisto/content/pull/30601/commits/cc59bddcbcfddffbc2e558a72a565654b0bfa5d1)